packages/react/src/components/ExpandCollapsePanel/index.css

```diff
@@ -31,3 +31,9 @@
   --expandcollapse-trigger-color: var(--white);
   --expandcollapse-panel-color: var(--accent-light);
 }
+
+@media (prefers-reduced-motion: reduce) {
+  .ExpandCollapse__panel {
+    animation: unset !important;
```

Comment:
`unset` behaves like `inherit` in some instances, so we should implicitly set it to `none`.